ci/glci/model.py

```diff
@@ -480,3 +480,13 @@ def feature_by_name(feature_name: str):
         if feature.name == feature_name:
             return feature
     raise ValueError(feature_name)
+
+
+@dataclasses.dataclass(frozen=True)
+class OpenRC:
+    auth_url: str = os.environ['OS_AUTH_URL']
```

Comment:
please do not consume values from ENVIRONMENT here.

If at all, this should be done at the pipeline-step script. However, I would prefer that configuration be passed from `cicd.yaml`

Reply:
Ok, we do not default them via enviroment. How about read the env in the `main()` function? This will be called only for dev/testing purposes.